docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+   * - `7.7. The yield statement <https://docs.python.org/3/reference/simple_stmts.html#the-yield-statement>`_
+     - Not Supported
+     -
+   * - `7.8. The raise statement <https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement>`_
+     - Partially Supported
+     - Message is not customized, ... from statements silently do nothing
+   * - `7.9. The break statement <https://docs.python.org/3/reference/simple_stmts.html#the-break-statement>`_
+     - Supported
+     - Some other features of returning (e.g. behavior with try..finally) are unsupported
+   * - `7.10. The continue statement <https://docs.python.org/3/reference/simple_stmts.html#the-continue-statement>`_
+     - Supported
+     - Some other features of returning (e.g. behavior with try..finally) are unsupported
+   * - `7.11. The import statement <https://docs.python.org/3/reference/simple_stmts.html#the-import-statement>`_
+     - Not Supported
+     -
+   * - `7.11.1. Future statements <https://docs.python.org/3/reference/simple_stmts.html#future-statements>`_
+     - Not Supported
+     -
+   * - `7.12. The global statement <https://docs.python.org/3/reference/simple_stmts.html#the-global-statement>`_
+     - Not Supported
+     -
+   * - `7.13. The nonlocal statement <https://docs.python.org/3/reference/simple_stmts.html#the-nonlocal-statement>`_
+     - Not Supported
+     -
+   * - `8. Compound statements <https://docs.python.org/3/reference/compound_stmts.html#>`_
+     - Irrelevant
+     -
+   * - `8.1. The if statement <https://docs.python.org/3/reference/compound_stmts.html#the-if-statement>`_
+     - Partially Supported
+     - String parser doesn't like semicolons
+   * - `8.2. The while statement <https://docs.python.org/3/reference/compound_stmts.html#the-while-statement>`_
+     - Partially Supported
+     - while..else doesn't work (and probably shouldn't), but we should fix the error
+   * - `8.3. The for statement <https://docs.python.org/3/reference/compound_stmts.html#the-for-statement>`_
+     - Partially Supported
+     - Document what can be iterated over, same issues as with while..else
+   * - `8.4. The try statement <https://docs.python.org/3/reference/compound_stmts.html#the-try-statement>`_
+     - Not Supported
+     -
+   * - `8.5. The with statement <https://docs.python.org/3/reference/compound_stmts.html#the-with-statement>`_
+     - Not Supported
+     -
+   * - `8.6. Function definitions <https://docs.python.org/3/reference/compound_stmts.html#function-definitions>`_
+     - Not Supported
+     - These are kind of supported for symbolic_script, but closures cannot be called and cannot be created in the Python frontend
+   * - `8.7. Class definitions <https://docs.python.org/3/reference/compound_stmts.html#class-definitions>`_
+     - Not Supported
+     - We need to make clear what is supported in Python land vs inside the TorchScript language
```

Comment:
Rephrase: `We need to make clear what is supported in Python land vs inside the TorchScript language` to be user-facing. I would say something like, `nn.Modules` are supported, support for other classes is experimental